### DISCUSSION TITLE
Forget error on backrest 0.15.0

### DISCUSSION THREAD

**Discussion by PFedya8** [garethgeorge/backrest#284] [Q&A] [answered]
Hello! I have next config for backup 
```
{
  "id": "plantest",
  "repo": "backup",
  "paths": [
    "/home/media"
  ],
  "excludes": [],
  "iexcludes": [],
  "cron": "30 0 * * *",
  "retention": {
    "policyKeepLastN": 10
  }
}
```
Backup is working fine, but retention policy doesn't work and every time I have next error `error: forget: get snapshots for repo backup: command "/root/.local/share/backrest/restic-0.16.4 forget --json -o sftp.args=-oBatchMode=yes --tag plan:plantest --group-by tag" failed: command output is not valid JSON: unexpected end of JSON input` 
How can I fix it? 
Thanks in advance!

**Comment by garethgeorge** [comment]
Hey, this is typically caused by locking errors. Go ahead and click “view logs” to check the restic output and confirm. 

You can unlock the repo once with the “unlock now” button OR you can enable “Auto Unlock” in the repo settings to solve this more permanently (if you’re only using one device to backup to that location)

**Reply by garethgeorge** [reply]
Just to add on here: I think there is a regression in the latest release re: command error formatting, typically Backrest should be including more details about the specific error that occurred without you needing to go to the logs. I'll aim to get that fixed in a patch release in the next week or so.

**Reply by PFedya8** [reply]
Thanks! I unlocked repo (with auto unlock and unlock now tried also) but it didn't help me still the same error. And I can't find view logs button where is it placed?

**Reply by garethgeorge** [reply]
View logs shows up right next to the operation that failed in the UI: 

![image](https://github.com/garethgeorge/backrest/assets/7906572/1a380803-9d4a-400d-a4d8-05c019252b28)

**Reply by PFedya8** [reply]
<img width="1124" alt="image" src="https://github.com/garethgeorge/backrest/assets/90137316/ea8331b1-a15a-4f07-b55c-ba903015c686">
I don't have it...

**Reply by garethgeorge** [reply]
Are you running Backrest 1.0.0?

**Reply by PFedya8** [reply]
No, 0.15.0

**Reply by garethgeorge** [reply, marked as answer]
Ah, got it. 0.15.0 has a known bug with forget operations, go ahead and update to at least 0.15.1 (or directly to 1.0.0).

**Reply by PFedya8** [reply]
oh, I see, then what is the best option to update it? I run it on ubuntu without docker

**Reply by garethgeorge** [reply]
You can simply go ahead and download the latest version and rerun the install.sh script. It will replace your current install and migrate configs.

**Reply by PFedya8** [reply]
Hey, it helped, error is gone, thank you so much!
